src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -69,6 +69,13 @@ public abstract class LinearTrainerBase<TTransformer, TModel> : TrainerEstimator
         {
         }
 
+        private protected LinearTrainerBase(IHostEnvironment env, string featureColumn, SchemaShape.Column labelColumn,
+            SchemaShape.Column weightColumn)
```

Comment:
why do you need this?

Reply:
I added this to account for the corresponding changes in line #1728-1730 

I saw some tests fail  saying "Weights" column not found.  The fix was to specify ` options.WeightColumn.IsExplicit` in the call below.  

`TrainerUtils.MakeR4ScalarWeightColumn(options.WeightColumn, options.WeightColumn.IsExplicit)`


---
In reply to: [248006972](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248006972) [](ancestors = 248006972)

Reply:
Added to #2100

---
In reply to: [248058147](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248058147) [](ancestors = 248058147,248006972)